content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Enable the Evaluate Artifacts Stage Plugin
+description: >
+  Enable the Evaluate Artifacts Stage plugin to add a stage to Armory Enterprise that makes it easy to evaluate SpEL expressions inside of Spinnaker™ artifacts.
+aliases:
+  - /docs/plugin-guide/plugin-evaluate-artifacts/
+---
+
+![Proprietary](/images/proprietary.svg)
+
+## Overview
+
+Armory’s Evaluate Artifacts plugin allows you to easily evaluate SpEL queries inside of any Spinnaker artifacts. Some artifact types and their associated stages, such as Kubernetes, support  using SpEL to read parameters from the Spinnaker context and inject them into an artifacts manifest at deploy time. Other stages, such as the Terraform Integration stage, do not support this out of the box. The Evaluate Artifacts Stage plugin adds supports for evaluating SpEL against any artifact within your Spinnaker pipeline.
+
+For information about how to use the stage, see [Use the Evaluate Artifacts Stage]({{< ref "evaluate-artifacts-plugin-use.md" >}}).
+
```

Comment:
Before `Setup`, what do you think of including a _Before you begin_  section that points to the OSS plugin use guide https://spinnaker.io/guides/user/plugins/ ? The first few sections cover terminology, but the latter half details how to install a plugin using Halyard.

Reply:
I have the directions for Halyard written up but haven't pushed them yet. But I'll add the link in the pre-reqs so people can reference the terminology.